### PR TITLE
Schema.rb.example updated

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Ah, i'm sorry, i think due to various PRs, this needs a rebase again? Thank you, @divyabaid16 !

**Comment by divyabaid16** [comment]
@jywarren I have rebased my PR. But I cannot see the Travis build :thinking:

**Comment by jywarren** [comment]
Seems like it's working now? Sometimes it takes a little while to start!

**Comment by jywarren** [comment]
OK, great, it passed. I'm requesting a couple more reviews just to be sure!

**Comment by gauravano** [comment]
Hey @divyabaid16, why are you deleting the migration?

**Comment by divyabaid16** [comment]
I deleted the migration because when it was present and after executing the db setup command, it was showing the error that versions table already exists. Can't be duplicated. So in order to remove that I deleted the migration. Travis was failing.

**Comment by kaustubh-nair** [comment]
Hi @divyabaid16 but if you delete the migration, the versions table will not be created while running `rails db:migrate` . Don't you think it's safer to keep the migration file?

**Comment by divyabaid16** [comment]
Ya, that is true.
So let me just add it again.

**Comment by SidharthBansal** [comment]
How is it going? Need any help on this?

**Comment by divyabaid16** [comment]
I have done this one. If any change is required I will make it.
Thanks!

File: db/migrate/20190715092943_create_versions.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class CreateVersions < ActiveRecord::Migration[5.2]
   TEXT_BYTES = 1_073_741_823
 
   def change
-    create_table :versions, { options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci" } do |t|
```
Comment (alaxalves) [inline db/migrate/20190715092943_create_versions.rb]: We need to enforce UTF-8, this is required by Rails 6. I think we should keep this.

**Comment by sashadev-sky** [comment]
@alaxalves @divyabaid16 I was just wondering, why do we keep a `schema.rb.example` file if we check in `schema.rb`. Do we need this at all?

**Comment by alaxalves** [comment]
> @alaxalves @divyabaid16 I was just wondering, why do we keep a `schema.rb.example` file if we check in `schema.rb`. Do we need this at all?

I have questioned this myself. But I imagined that we keep an example file in case someone wants to "lift" a MK instance. Or if we ourselves maintain a different version of MK. If this is not true we can either erase the `schema.rb` file or the example file

**Comment by sashadev-sky** [comment]
I was thinking maybe lets delete it? It's autogenerated. Its almost like checking in a `yarn.lock.example` right? And in that case it would be a 'copy' not so much an example @divyabaid16 what are thoughts

**Comment by alaxalves** [comment]
> I was thinking maybe lets delete it? It's autogenerated. Its almost like checking in a `yarn.lock.example` right? And in that case it would be a 'copy' not so much an example @divyabaid16 what are thoughts

Totally agree with this. It's less overhead!

**Comment by divyabaid16** [comment]
Ya, it sounds good to me too.

**Comment by sashadev-sky** [comment]
@divyabaid16 ok awesome! So do you want to delete it here? I dont know the other hald of the work you were doing here with versions what was that for?

I could also just delete it in my PR #959

**Comment by jywarren** [comment]
Just noting that we do not track it in plots2 -- only the example, in case
people want to build a database without running through the migrations one
by one. In case that helps the discussion!

**Comment by sashadev-sky** [comment]
@jywarren i'm confused, what is the benefit then to checking in an exact duplicate of the `schema.rb` instead of the file itself

**Pull request closed without merging**